### ISSUE TITLE
3080ti Supreme X 77mh speed

### ISSUE THREAD

**Issue by M3dic** [trexminer/T-Rex#806]
I have problem with setting this card. Always 77mh/s and core 900MHz `80-96pl` and `1000-1500 mclock`. Nothing change after `pl` and `core clock` cahgned . I cant overclock this card in no way?
Any suggestions?

**Comment by pdieppa** [comment]
I came across this video from Red Fox Crypto
https://www.youtube.com/watch?v=ok2WP1_BPqg

**Comment by chaosclarity84** [comment]
your core is too low, it's killing the hashrate.

**Comment by M3dic** [comment]
@ chaos clarity 84 Yes but nothing can change it. Neither form trex miner or from MSI Afterburner

**Comment by M3dic** [comment]
![image](https://user-images.githubusercontent.com/34350927/140471278-5d8c2cfe-a2e6-4901-af55-6c39c90b2ab9.png)

**Comment by vchixaro** [comment]
I too have struggled with MSI Suprim and TRIO.
On the Suprim I was able do to 85mh, but I had to set memory clock offset for underclocking (-50) and gpu core at 1200. Anything higher puts the card core is speed at 800-900mhz. It seems that MSI card has some kind of power limit for ethash. Even when set a 110%.

**Comment by Mollucans** [comment]
anyone tried to flash bios 3080ti suprim to another brand ? cross flash
i got same problem also

**Comment by vchixaro** [comment]
Yes, I had also a MSI TRIO. Flashed both Suprim and Trio with the EVGA FTW3 BIOS.
https://www.techpowerup.com/vgabios/235920/evga-rtx3080ti-12288-210719
All cards doing 90mh now.
The only issue is the power consumption report. Its higher, but the actual consumption is ok. This may be due to the diferent power delivery. MSI cards where pulling 150W from one PCIe power and only 20/30 from the others, and this is causing power limitation while mining.

**Comment by stefgina** [comment]
The MSI SuprimX limit its clock, because of a power threshold Bios missmatch. It turns out that when PCIE maxes out of power and plug 1 8pin also maxes out, the rest of the two plugs (plug2 8pin, plug3 8pin) cant max out their watts.

So for anyone want to run the card with its own Bios, send a ticket to MSI reporting the issue to speedup the BIOS update process.

If cross bios doesnt bother you, the EVGA FTW3 3080ti Bios works fine as reported 89-90mhs . Rog Strix Bios meshes up the LEDs, the power consumption metrics and also the fan RPMS, but reporting 91-92mhs.
